### ISSUE TITLE
Abstract terrain scale out of CanvasController

### ISSUE THREAD

**Issue by n8rzz** [openscope/openscope#798]
This could be achieved with HTML or a simple SVG.  Either way, it should not be part of the CanvasController (thus requiring re-rendering with text every frame).

**Comment by sam-irl** [comment]
I believe this should supercede #595 if I'm understanding correctly?

**Comment by n8rzz** [comment]
@sam-irl good catch. No, I used the wrong word. That issue is still a thing, but should be tackled after #799 is merged.
